files/image_config/bash/bash.bashrc

```diff
@@ -56,3 +56,12 @@ fi
 
 # Automatically log out console ttyS* sessions after 15 minutes of inactivity
 tty | grep ttyS >/dev/null && TMOUT=900
+
+# if SSH_TARGET_CONSOLE_LINE was set, attach to console line interactive cli directly
+if [ -n "$SSH_TARGET_CONSOLE_LINE" ]; then
```

Comment:
i think we need to make it optional. for example, if the device is not c0 device, then we must not act on this variable even if it is not empty.

also, we need to verify the content this variable to make sure it is valid. this potentially allow hack to modify the variable on a shell.

Reply:
For the first comment:
I think we can made a build feature flag for console switch feature. 
In my view, this feature is not platform dependent. e.g. A E1031 can be both a bare management switch (without console hub) and console switch(plugged a console hub).
Only user can decide if they want to use it or not. Let me know your thought...

For the second comment:
Agree. We should only accept integer here. I will update.

Reply:
it is hard to make it as a build flag. imo, it should depends on the device role, for example mgmttorrouter, or in config db, we can add a switch metadata to see where console switch is enabled or not.

Reply:
I prefer use config db. Add a key to enable/disable console switch features. And user can also use cli `config console {on/off}` to config that.
For minigraph user, we can simply enable console switch by judging if there is a console port.

If we do so, we can keep this code for a while. I will raise a new PR to update the script after config part done.